src/com/itschool/session4/homework/challenge02/ReverseSign.java

```diff
@@ -0,0 +1,22 @@
+package com.itschool.session4.homework.challenge02;
+import java.util.Scanner;
+
+public class ReverseSign {
+    public static void main(String[] args) {
+
+        //creaza un nou obiect scanner
+        Scanner scanner = new Scanner(System.in);
+
+        //Afiseza promptul
+        System.out.println("Introdu numarul:");
+        int numar1 = scanner.nextInt();
+
+        //Afiseaza numarul cu minus
+
+        int semnMinus = -numar1;
+        System.out.println("Numarul negativ este:"+ semnMinus);
+
+
+
```

Comment:
some extra empty lines here that can be removed.

Reply:
thx for observations